docs/content/en/docs/use-cases/hpa.md

```diff
@@ -4,41 +4,143 @@ description: Using the HorizontalPodAutoscaler
-spec:
-  scaleTargetRef:
-    apiVersion: apps/v1
-    kind: Deployment
-    name: podtato-head-entry
-  minReplicas: 1
-  maxReplicas: 10
-  metrics:
-    - type: Object
-      object:
-        metric:
-          name: keptnmetric-sample
-        describedObject:
-          apiVersion: metrics.keptn.sh/v1beta1
-          kind: KeptnMetric
-          name: keptnmetric-sample
-        target:
-          type: Value
-          value: "10"
+## Scaling Workloads based on Keptn metrics
+
+Kubernetes provides many built-in capabilities to ensure
+running enough replicas in order to meet the current demand of the Workloads.
+One of these is the
+[HorizontalPodAutoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
+(HPA).
+
+HPA can make use of the Keptn Metrics
+[custom API](https://kubernetes.io/docs/reference/external-api/custom-metrics.v1beta2/)
+to scale the number of replicas of the Workloads based on the current
+load.
+It does this by using metrics such as CPU throttling, memory consumption, or response time.
+
+### Installation of Keptn Metrics Operator
+
+To use HPA with the custom metrics API,
+Keptn Metrics Operator must be installed on the cluster.
+For more information about installation please refer to the official
+[installation guide](../installation/_index.md).
+
+> **Note**
+The Keptn Lifecycle Operator does not need to be installed for this use-case.
+
+### Installation of metrics provider (optional)
+
+If you do not yet have a metrics provider installed on your cluster yet, please do so.
+
+For this tutorial we are going to use [Prometheus](https://prometheus.io/).
+For more information about how to install Prometheus into your cluster, please
+refer to the [official Prometheus documentation](https://prometheus.io/docs/prometheus/latest/installation/).
+
+### Deploy sample application
+
+As a first step, we need to deploy our application to the cluster.
+For this we are going to
+use a single service `podtato-head` application.
+
+{{< embed path="/docs/assets/hpa/sample-app.yaml" >}}
+
+Please create a `podtato-kubectl` namespace and apply the above manifest
+to your cluster and continue with the next steps.
+After applying, please make sure that the application is up and running:
+
+```shell
+$ kubectl get pods -n podtato-kubectl
+podtato-head-entry-58d6485d9b-ld9x2         1/1     Running     (2m ago)
+```
+
+### Create KeptnMetric and KeptnMetricsProvider custom resources
+
+To be able to react on the metrics of our application, we need to create
+`KeptnMetrics` and `KeptnMetricsProvider` custom resources.
+These metrics are
+exposed via custom metrics API, what gives us the possibility to configure HPA
+to react on the values of these metrics:
+
+{{< embed path="/docs/assets/hpa/keptnmetric.yaml" >}}
+
+For more information about the `KeptnMetric` and `KeptnMetricsProvider` custom resources,
+please refer to the official [CRD documentation](../crd-ref/metrics/v1beta1/_index.md).
```

Comment:
you don't need to add the index file if you don't link to a specific anchor inside the file
```suggestion
please refer to the [CRD documentation](../crd-ref/metrics/v1beta1/).
```